src/ConcreteOperations/minkowski_sum.jl

```diff
@@ -389,9 +395,17 @@ function minkowski_sum(P1::VPolytope, P2::VPolytope;
 
     vlist1 = _vertices_list(P1, backend)
     vlist2 = _vertices_list(P2, backend)
+    Vout = _minkowski_sum_vrep_nd(vlist1, vlist2, apply_convex_hull=apply_convex_hull, backend=backend, solver=solver)
+    return VPolytope(Vout)
+end
+
+function _minkowski_sum_vrep_nd(vlist1::Vector{VT}, vlist2::Vector{VT};
```

Comment:
What if the vectors have mixed types?

Reply:
i think that use case is quite uncommon and we can leave the conversion to the user. the new thing that this brings is that before we hardcoded to `Vector{N}` and now it uses `VT`. im not even sure if this makes sense in general because if `VT` is a SingleEntryVector, their sum need not be the of that type.

Reply:
A vertex of type `SingleEntryVector` would be strange. The only case I see is an `SVector` mixed with a normal `Vector`.
But you are right that this was not possible before as well, so this is already a step forward.